avionics/sensors.cpp

```diff
@@ -303,16 +304,22 @@ void logData(unsigned long *timestamp, float *battery_voltage, float acc_data[],
 }
 
 
-void calculateValues(float acc_data[], float bar_data[], float* abs_accel,
-                    float* prev_altitude, float* altitude, float* delta_altitude)
-{
-    // *abs_accel = sqrtf(powf(acc_data[0], 2) + powf(acc_data[1], 2) + powf(acc_data[2]), 2);
-    // *prev_altitude = *altitude;
-    // *altitude = 44330.0 * (1 - powf(bar_data[0] / BASELINE_PRESSURE, 1 / 5.255));
-    // *delta_altitude = altitude - prev_altitude;
-}
-
+/*
+* @brief replaces oldest value in the average pressure set with a new peice of pressure data
+*  if values are 0 it replaces them with data. This allows for the average to be non jank imediatly
+* @param float* average_set - the working array of pressure data used to calculate the average pressure. This is mutated by replacing the oldest data as new readings come in.
+* @param float data - the new data to add to the set
+* @return void
+*/
+void addToPressureSet(float* average_set, float data){
```

Comment:
Why is this in sensors.cpp and not calculations.cpp?

Reply:
Makes sense to move, although there's another thing this reminds me of - as it's currently implemented, addToPressureSet and calculatePressureAverage are done outside of calculateValues. It might make more sense to fold those two into calculateValues